README.md

```diff
@@ -1,12 +1,46 @@
 # Self-healing Apache Zookeeper ECS cluster
 
 ## General information
-This repo contains terraform code and docker file which will allow you to create self-healing Apache Zookeeper cluster into AWS ECS
+The repository is a fork from https://github.com/dethmix/self-healing-zookeeper which the goal is to have a Zookeeper server that is self-healing and is deployed in AWS ECS, this has been further improved by removing the ECS autodiscovery feature as it creates complexity during deployment and automate the Zookeeper node enrollment.
```

Comment:
The repository is a fork from https://github.com/dethmix/self-healing-zookeeper. `self-healing-zookeeper` deploys each instance of the zookeeper cluster as a separate service. It then uses ECS cloud map to discover its peers in the cluster. 

With this repo we aimed to provide this with lesser number of moving parts. Instead of keeping the cluster nodes as part of separate services, we combined them into 1 single service. We used the underlying behavior of how nodes are scheduled in a service when there are multiple subnets.  If the service has multiple subnets, each replica that comes up for the service will attach to the subnet that has no instances in that subnet.